src/gremlin.py

```diff
@@ -1,14 +1,30 @@
 """Abstracts access to gremlin server through REST calls"""
 
+import json
+import logging
+import requests
+import daiquiri
 from gremlin_python.structure.io import graphsonV3d0
+from src.config import GREMLIN_DEFAULT_HOST, GREMLIN_DEFAULT_PORT, GREMLIN_SCHEME
 
-from gremlin_connect.gremlin_adapter import GremlinAdapter
-from src.config import GREMLIN_DEFAULT_HOST, GREMLIN_DEFAULT_PORT
+daiquiri.setup(level=logging.DEBUG)
```

Comment:
Can you read  this logging mode from config file so if we wanted to change to INFO then can change config without changing code.

Reply:
Sure. I can add a env variable to control the level.